types/responses/subscription.ts

```diff
@@ -89,6 +89,7 @@ export type SubscriptionCidrWhitelist = {
  * @param awsAccountId The AWS account ID (Optional)
  * @param vpcUid The VPC ID (Optional)
  * @param vpcCidr The VPC CIDR (Optional)
+ * @param vpcCidrs The multiple VPC CIDRs (Optional)
```

Comment:
Couldn't we use one parameter instead of these two? `vpcCidr`, `vpcCidrs`

Reply:
it's aligned with the Dev's decision - if only 1 CIDR then we return `vpcCidr`, if more than 1 then we return `vpcCidrs`